iphone/Classes/TiUIWebView.m

```diff
@@ -540,6 +540,13 @@ -(void)setHandlePlatformUrl_:(id)arg
     willHandleUrl = [TiUtils boolValue:arg];
 }
 
+-(void)setUserAgent_:(id)args
+{
+    ENSURE_TYPE(args, NSString);
```

Comment:
Use `[self proxy] replaceValue:args forKey:@"userAgent" notification: NO]` for retaining the proxy state.